src/backend/joanie/core/serializers.py

```diff
@@ -33,23 +198,69 @@ class Meta:
             "products",
         ]
 
+    def to_representation(self, instance):
+        """
+        When user is not authenticated, we return the cached representation,
+        otherwise we generate representation as well
+        """
+        code = self.context.get("code")
+        user = self.context.get("user", None)
```

Comment:
as the None default is implicit, why do you declare it explicitly here and not for "code" in the line above?

Reply:
Let's remove this explicit default value.